doc/whats_new/v0.23.rst

```diff
@@ -544,6 +544,7 @@ Changelog
 - |Enhancement| :class:`inspection.PartialDependenceDisplay` now exposes the
   deciles lines as attributes so they can be hidden or customized. :pr:`15785`
   by `Nicolas Hug`_
+>>>>>>> 3017ea8c30e82931895f4ec46c4a5a9d00ed67c2
```

Comment:
```suggestion

```
This is a relic of the diff: it produces the sphinx warning and the docstring error.

Reply:
Thanks, sorry I meant to resolve the rest of this last night, but got distracted and forgot to respond!